src/graphql/user/mongoose_graphql_muc_light_user_mutation.erl

```diff
@@ -0,0 +1,82 @@
+-module(mongoose_graphql_muc_light_user_mutation).
+
+-export([execute/4]).
+
+-ignore_xref([execute/4]).
+
+-include("../mongoose_graphql_types.hrl").
+
+-import(mongoose_graphql_helper, [make_error/2, format_result/2]).
+-import(mongoose_graphql_muc_light_helper, [make_room/1, make_ok_user/1, prepare_blocking_items/1]).
+
+execute(Ctx, _Obj, <<"createRoom">>, Args) ->
+    create_room(Ctx, Args);
+execute(Ctx, _Obj, <<"changeRoomConfiguration">>, Args) ->
+    change_room_config(Ctx, Args);
+execute(Ctx, _Obj, <<"inviteUser">>, Args) ->
+    invite_user(Ctx, Args);
+execute(Ctx, _Obj, <<"deleteRoom">>, Args) ->
+    delete_room(Ctx, Args);
+execute(Ctx, _Obj, <<"kickUser">>, Args) ->
+    kick_user(Ctx, Args);
+execute(Ctx, _Obj, <<"sendMessageToRoom">>, Args) ->
+    send_msg_to_room(Ctx, Args);
+execute(Ctx, _Obj, <<"setBlockingList">>, Args) ->
+    set_blocking_list(Ctx, Args).
+
+-spec create_room(map(), map()) -> {ok, map()} | {error, resolver_error()}.
+create_room(Ctx, #{<<"id">> := null} = Args) ->
+    create_room(Ctx, Args#{<<"id">> => <<>>});
```

Comment:
I still think it would be better not to treat `""` the same way as `null` in the GraphQL API. Not a must though.